src/crud/useMediaAttach.ts

```diff
@@ -28,21 +28,21 @@ export const useMediaAttach = (props: IProps) => {
   ) => {
     var tb = new TransformBuilder();
     var ops: Operation[] = [];
+    var mediaRI = {type: 'mediafile', id: mediaId};
+    if (related(memory.cache.query(q => q.findRecord(mediaRI)), 'passage') === passage)
```

Comment:
this looks funny b/c in theory, media.cache.query might return null if the record is not found and we are not testing for it.